src/PhpParser/Node/BetterNodeFinder.php

```diff
@@ -225,7 +225,7 @@ public function findFirstNonAnonymousClass(array $nodes): ?Node
 
     /**
      * @param Node|Node[] $nodes
-     * @param callable(Node $node): bool $filter
+     * @param callable(Node $filter): bool $filter
```

Comment:
This "fix" looks like a workaround

Reply:
Yes

Reply:
that's possibly due to cs fixer:

```diff
-     * @param Node|Node[] $nodes
-     * @param callable(Node $node): bool $filter
+     * @param Node|Node[] $filters
+     * @param callable(Node $filter): bool $filter
      */
     public function findFirst(Node | array $nodes, callable $filter): ?Node
     {
    ----------- end diff -----------


Applied checkers:

 * PhpCsFixer\Fixer\Operator\NotOperatorWithSuccessorSpaceFixer
 * PhpCsFixer\Fixer\Operator\UnaryOperatorSpacesFixer
 * PhpCsFixer\Fixer\Semicolon\NoSinglelineWhitespaceBeforeSemicolonsFixer
 * Symplify\CodingStandard\Fixer\Commenting\ParamReturnAndVarTagMalformsFixer
```

Reply:
Ok,  it seems because of `Symplify\CodingStandard\Fixer\Commenting\ParamReturnAndVarTagMalformsFixer`